code/game/objects/items/weapons/design_disks/excelsior.dm

```diff
@@ -0,0 +1,157 @@
+		/datum/design/autolathe/part/igniter,						//regular parts
+		/datum/design/autolathe/part/signaler,
+		/datum/design/autolathe/part/sensor_prox,
+		/datum/design/autolathe/cell/large/excelsior,				//power cells
+		/datum/design/autolathe/cell/medium/excelsior,
+		/datum/design/autolathe/cell/small/excelsior,
+		/datum/design/autolathe/prosthesis/excelsior/l_arm,         //prostheses
+		/datum/design/autolathe/prosthesis/excelsior/r_arm,
+		/datum/design/autolathe/prosthesis/excelsior/l_leg,
+		/datum/design/autolathe/prosthesis/excelsior/r_leg,
+		/datum/design/autolathe/device/implanter,					//misc
+		/datum/design/autolathe/device/propaganda_chip,
+	)
+
+/obj/item/weapon/computer_hardware/hard_drive/portable/design/excelsior/weapons
+	disk_name = "Excelsior Means of Revolution"
+	desc = "The back has a machine etching: \"We stand for organized terror - this should be frankly admitted. Terror is an absolute necessity during times of revolution.\""
+	spawn_blacklisted = TRUE
+	license = -1
+	designs = list(
+		/datum/design/autolathe/gun/makarov,
+		/datum/design/autolathe/gun/drozd,
+		/datum/design/autolathe/gun/vintorez,
+		/datum/design/autolathe/gun/boltgun,
+		/datum/design/autolathe/gun/ak47,
+		/datum/design/autolathe/ammo/magazine_pistol,				//makarov ammo
+		/datum/design/autolathe/ammo/magazine_pistol/rubber,
+		/datum/design/autolathe/ammo/pistol_ammobox,
+		/datum/design/autolathe/ammo/pistol_ammobox/rubber,
+		/datum/design/autolathe/ammo/msmg,							//drozd ammo
+		/datum/design/autolathe/ammo/msmg/rubber,
+		/datum/design/autolathe/ammo/magnum_ammobox,
+		/datum/design/autolathe/ammo/magnum_ammobox/rubber,
+		/datum/design/autolathe/ammo/srifle,						//vintorez ammo
+		/datum/design/autolathe/ammo/srifle/rubber,
+		/datum/design/autolathe/ammo/srifle_ammobox,
+		/datum/design/autolathe/ammo/srifle_ammobox/rubber,
+		/datum/design/autolathe/ammo/sl_lrifle,						//boltgun ammo
+		/datum/design/autolathe/ammo/lrifle_ammobox_small,
+		/datum/design/autolathe/ammo/lrifle_ammobox_small/rubber,
+		/datum/design/autolathe/ammo/lrifle,						//AK ammo
+		/datum/design/autolathe/ammo/lrifle/rubber,
+		/datum/design/autolathe/ammo/lrifle_ammobox,
+		/datum/design/autolathe/sec/silencer,						//misc
+		/datum/design/autolathe/clothing/excelsior_armor,
+		/datum/design/autolathe/device/excelbaton,					//security
+		/datum/design/autolathe/device/excelsiormine,
+		/datum/design/autolathe/sec/beartrap,
+	)
+
+/obj/item/weapon/computer_hardware/hard_drive/portable/design/excelsior/drozd
+	disk_name = "Excelsior - .40 Drozd SMG"
+	desc = {"The back has a machine etching:\n \
+	\"Nobody is to be blamed for being born a slave; \
+	but a slave who not only eschews a striving for freedom but justifies and eulogies his slavery - \
+	such a slave is a lickspittle and a boor, who arouses a legitimate feeling of indignation, contempt, and loathing.\""}
+	spawn_tags = SPAWN_TAG_DESIGN_ADVANCED_EXCELSIOR
+	rarity_value = 50
+	license = 14
```

Comment:
Excelsior disks having DRM goes directly against what they stand for, as DRM exists to perpetuate private (intellectual) property. It's not like giving these disks a license point cost will actually stop anything, either, because you can always clone the disk. I can get why you'd want to try and limit how many Excel guns can get around the ship, but honestly, I think it's a lost cause.

Reply:
I agree with this.

Reply:
Resolved.